client/src/components/dashboard/leagueTabs/leagueTabData.js

```diff
@@ -80,13 +80,13 @@ const settingsLinks = [
 		label: 'AddStaff',
 		icon: <AddPersonIcon />,
 		url: LINK.SETTINGS_ADD_STAFF_FORM
-	},
-	{
-		description: 'Delete your league.',
-		label: 'DeleteLeague',
-		icon: <DeleteForeverIcon />,
-		url: LINK.SETTINGS_DELETE_LEAGUE
 	}
+//	{
```

Comment:
Just commented out until delete was explained.

These lines can be completely removed.